### ISSUE TITLE
Problem when using Material Widgets like Cards with ListWheelScrollview

### ISSUE THREAD

**Issue by Loopex2019** [flutter/flutter#49052]
Hi i have a Problem when using any Material widgets which usually supports the elevation parameter
like the `Card`  or even the `Material` Widget. And the problem is that when using these kind of widgets with the `ListWhellScrollView` , it prevents it from scrolling and lays out the cards (as an example) on top of each other . Here is my Code : 

```
import 'package:flutter/material.dart';
import 'package:superellipse_shape/superellipse_shape.dart';

class HalfCircleExample extends StatelessWidget {
  Widget _buildItem() {
    return Center(
      child: Container(
        width: 50,
        height: 50,
        child: Card(
          shape: SuperellipseShape(
            borderRadius: BorderRadius.circular(40),
          ),
        ),
      ),
    );
  }

  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('WheelExample'),
      ),
      body: SafeArea(
        child: Center(
          child: ListWheelScrollView(
            itemExtent: 60,
            children: List.generate(
              10,
              (int i) => _buildItem(),
            ),
          ),
        ),
      ),
    );
  }
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Flutter Demo',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: HalfCircleExample(),
    );
  }
}

void main() => runApp(MyApp());
```

and here is an image of the output  : 

![Screenshot_20200117-161800](https://user-images.githubusercontent.com/49901727/72619266-96938b80-3945-11ea-82b3-f6360cefb9b9.jpg)

and here is my flutter doctor : 

```
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, v1.12.13+hotfix.5, on Microsoft Windows [Version 10.0.17763.914], locale en-US)

[√] Android toolchain - develop for Android devices (Android SDK version 29.0.0-rc1)
[√] Android Studio (version 3.3)
[√] VS Code (version 1.41.1)
[√] Connected device (1 available)

• No issues found!
```

**Comment by VladyslavBondarenko** [comment]
Hi @Loopex2019 
It can be fixed with next release of stable version.
You may consider to switch to dev channel, it was already fixed there and your sample is working properly on dev.
Closing the issue
If you disagree please write in the comments and I will reopen it. 
Thank you

**Comment by Loopex2019** [comment]
I switched to the `dev` channel but it showed me nothing when using Material Widgets. It worked normally with other widgets.

**Comment by Loopex2019** [comment]
I think we should reopen it just to notify the flutter team for the next stable channel release about working on that issue

**Comment by VladyslavBondarenko** [comment]
Fix is already in stable (Tried with 1.17.1). Closing
